src/createWidgetBase.ts

```diff
@@ -143,6 +149,24 @@ function manageDetachedChildren(instance: Widget<WidgetProperties>): void {
 	});
 }
 
+function bindFunctionProperties(instance: Widget<WidgetProperties>, properties: WidgetProperties) {
+	Object.keys(properties).forEach((propertyKey) => {
+		const func = properties[propertyKey];
+		const bind = properties.bind;
+
+		if (typeof func === 'function') {
```

Comment:
I guess I feel like some of this belongs in dojo/core probably, so that we're not reimplementing binding throughout Dojo 2, though I realize this is pretty specific.